src/plugins/common/src/traits/remove_resource.rs

```diff
@@ -0,0 +1,35 @@
+use bevy::prelude::*;
+
+pub trait RemoveResource
+where
+	Self: Resource + Sized,
+{
+	fn remove(mut commands: Commands) {
+		commands.remove_resource::<Self>();
+	}
+}
+
+impl<T> RemoveResource for T where T: Resource {}
+
+#[cfg(test)]
+mod tests {
+	use super::*;
+	use bevy::ecs::system::RunSystemOnce;
+
+	#[derive(Resource, Default, Debug, PartialEq)]
+	struct _Resource;
+
+	fn setup() -> App {
+		App::new()
+	}
+
+	#[test]
+	fn init_resource() {
+		let mut app = setup();
+		app.init_resource::<_Resource>();
+
+		app.world_mut().run_system_once(_Resource::remove);
+
+		assert_eq!(None, app.world().get_resource::<_Resource>());
+	}
+}
```

Comment:
I don know, how I could make those tests fail. This would be different for de-spawning entities, where it could easily happen to produce a fatal error, but here I do no see any value to those tests. However, I have fixed the test name.